incubator/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaSessionFactory.java

```diff
@@ -912,6 +914,8 @@ public final class KafkaSignalStream
         private long replyAck;
         private int replyMax;
         private long reconnectAt;
+        private int sessionSignalDecodeSlot = NO_SLOT;
+        private int sessionSignalDecodeSlotOffset;
```

Comment:
Rename to `decodeSlot` and `decodeSlotOffset` as we are in scope of `KafkaSignalStream`.